src/controllers/index.js

```diff
@@ -1,10 +1,12 @@
 const express = require('express');
 const login = require('./login');
-
+const adminHomePage = require('./adminHomePage');
+const { authCheck } = require('./middleware');
 
 const router = express.Router();
 router.get('/admin/login', login.get);
 router.post('/admin/login', login.post);
-
-
+router.use(authCheck);
+router.get('/admin',adminHomePage.get);
```

Comment:
Fix indentation, and while you're doing it add a newline after requiring express, to separate module requires from file requires.